src/backend/dx11/src/lib.rs

```diff
@@ -723,6 +741,86 @@ impl hal::queue::RawCommandQueue<Backend> for CommandQueue {
 
 }
 
+#[derive(Derivative, Clone)]
+#[derivative(Debug)]
+struct AttachmentClear {
+    subpass_id: Option<pass::SubpassId>,
+    #[derivative(Debug="ignore")]
+    value: Option<command::ClearValueRaw>,
```

Comment:
I think having `Debug` implemented for `ClearValueRaw` (manually) would be handy for other backends and even users

Reply:
Agree. How would it handle [the union](https://github.com/gfx-rs/gfx/blob/master/src/hal/src/command/raw.rs#L45)? just print out all the variants?

Reply:
Just print out the `u32` one